psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -256,29 +367,44 @@ private static void createBuses(PsseRawModel psseModel, ContainersMapping contai
             // create bus
             createBus(psseBus, voltageLevel);
 
+            busNumToPsseBus.put(psseBus.getI(), psseBus);
+
         }
     }
 
     private static void createLine(PsseNonTransformerBranch psseLine, ContainersMapping containerMapping, PerUnitContext perUnitContext, Network network) {
         String id = "L-" + psseLine.getI() + "-" + psseLine.getJ() + "-" + psseLine.getCkt();
+
+        //build a unique name
+        String idTmp = id;
+        int i = 2;
+        while (network.getLine(idTmp) != null) {
+            idTmp = id + "-" + i;
+            i++;
+        }
+
+        id = idTmp;
+
         String bus1Id = getBusId(psseLine.getI());
         String bus2Id = getBusId(psseLine.getJ());
         String voltageLevel1Id = containerMapping.getVoltageLevelId(psseLine.getI());
         String voltageLevel2Id = containerMapping.getVoltageLevelId(psseLine.getJ());
         VoltageLevel voltageLevel2 = network.getVoltageLevel(voltageLevel2Id);
         double zb = Math.pow(voltageLevel2.getNominalV(), 2) / perUnitContext.getSb();
+        double x = psseLine.getX();
```

Comment:
I don't understand why do you extract `x`. As it's not used, it's not really necessary. Why `x` but not `r` then?

Reply:
OK, removed.